apps/web/src/actions/evaluations/runBatch.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  DatasetsRepository,
+  DocumentVersionsRepository,
+  EvaluationsRepository,
+} from '@latitude-data/core/repositories'
+import { setupJobs } from '@latitude-data/jobs'
+import { nanoid } from 'nanoid'
+import { z } from 'zod'
+
+import { authProcedure } from '../procedures'
+
+export const runBatchAction = authProcedure
+  .createServerAction()
+  .input(
+    z.object({
+      datasetId: z.number(),
+      projectId: z.number(),
+      documentUuid: z.string(),
+      commitUuid: z.string(),
+      runCount: z.number(),
+      offset: z.number().optional().default(0),
+      parameters: z.record(z.number()).optional(),
```

Comment:
Why is this a record of numbers?

Reply:
it maps parameters to the index column of rows in a csv, are you doing it differently in frontend?

Reply:
But you also need the name of the parameters in the documents no? It's a map between document params and csv headers or I'm missunderstanding?
<img width="611" alt="image" src="https://github.com/user-attachments/assets/89aabefd-d363-4d1b-bcc4-dbeca1f9f04a">

Reply:
yes the key is the parameter name and the value is the column index

Reply:
so in this case let's assume the csv has last_name and name in columns 3 and 4, the parameters map would be

{ last_name: 3, name: 4} the keys being the document parameters and the values being the csv collumn indeces

Reply:
It's true, headers is an array 👍